src/middleware/default.ts

```diff
@@ -6,6 +6,10 @@ export const defaultMiddleware = async (c: Context, next: Function) => {
     const url = new URL(c.req.url)
     return url.searchParams.get(key)
   }
+  c.req.params = c.req.param // alias
```

Comment:
Why we just make `ctx.req.params` at first place to avoid this alias?

Reply:
I see, that is better to make the alias after this: https://github.com/yusukebe/hono/pull/62/files#diff-fef0c99deaa99c63dde9a9af2451ae5281a68e92c338218d644a08b358878245R131

Reply:
Why we have both `params` and `param` in a `Request`, is possible to make it only `params` or `param`?

Reply:
Yes, we make it `param` only because `params` is just for backward compatibility and that we have both may cause confusion.